docs: hepa door rollers 2

jwwojak opened the pull request:
# Overview

As my previous PR exploded, this is HEPA Rollers 2: Electric Boogaloo. This PR does 2 things:

- It updates the HEPA module instruction manual to include the new rollers (images and text).

- It removes the logo image from the front page (`index.md`)

See [RTC-763](https://opentrons.atlassian.net/browse/RTC-763).

## Test Plan and Hands on Testing

Did I break the build? Hopefully not.

## Changelog

New renders and illustrations for the manual:

- Cover
- Labeled parts image
- Parts list
- Door roller section 

What doesn't change: images in the setup procedure. Let's bank our Jackson image requests for the vacuum manifold. I really don't want to ask him for revisions to these, they're still clear and valid.

## Review requests

Make sure I'm not destroying anything.

## Risk assessment

- High because it's my PR (well-intentioned, but chaotic good)
- Low because it's docs.



[RTC-763]: https://opentrons.atlassian.net/browse/RTC-763?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

jwwojak committed:
Door roller and image updates

- Adds section on door rollers

- Updates images to include module showing door rollers

- Images in procedures not updated. They're clear enough that people understand. Don't have the heart to ask Jackson to change these. Save image request capital for the vacuum manifold.

jwwojak committed:
Fixing cover image

ecormany commented on docs/hepa-uv/docs/images/hepa-cover-hero.svg:
This isn't an SVG at all. Should be saved as a PNG.

jwwojak replied:
Re-saving as .png.

ecormany commented on docs/hepa-uv/docs/images/hepa-features-labeled.svg:
This at least has some vector elements, but is probably also better off as a PNG.

jwwojak replied:
Resaving as .png.

ecormany commented on docs/hepa-uv/docs/product-specs.md:
```diff
@@ -83,3 +83,13 @@ The power consumption specifications are measured with the UV lights and fan on.
+
+## Door Protection Rollers
+
+With early model HEPA/UV modules, fully opening the robot's front door meant scraping it across the top of that module. Eventually, the edge of the HEPA module would scratch and leave scuff marks on the surface of the door.
```
Even though I think this was happening a lot, let's soften our modal verbs:
```suggestion
With early model HEPA/UV Modules, fully opening the robot's front door could cause it to scrape against the top of the module. The edge of the HEPA/UV Module could scratch and leave scuff marks on the surface of the door.
```

ecormany commented on docs/hepa-uv/docs/product-specs.md:
```diff
@@ -83,3 +83,13 @@ The power consumption specifications are measured with the UV lights and fan on.
+
+With early model HEPA/UV modules, fully opening the robot's front door meant scraping it across the top of that module. Eventually, the edge of the HEPA module would scratch and leave scuff marks on the surface of the door.
+
+![Module without door rollers](images/hepa-without-rollers.png)
```
Optional: move this after the next paragraph and do a `<figure class="side-by-side">` for a nice visual comparison.

jwwojak replied:
Yes, let's make it side by side.

ecormany reviewed (commented):
Couple comments. Let me know if you need help sourcing or reformatting the image files.

jwwojak committed:
Update docs/hepa-uv/docs/product-specs.md

Thank you for the suggestion. Applying it now.

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>

jwwojak committed:
Image file format and door roller text changes

- Removed converted SVGs and replaced with true PNG images.

- Make model comparison a side by side image. Removed rounded arrows. Maybe those aren't needed and cause clutter.

- Tweak door roller text

ecormany commented on docs/hepa-uv/docs/product-specs.md:
```diff
@@ -86,10 +86,6 @@ The power consumption specifications are measured with the UV lights and fan on.
-HEPA modules manufactured on or after July 2025 include three rollers mounted along its top-front edge. These rollers prevent the HEPA module from damaging the front door when you open it.
-
-![Module with door rollers](images/hepa-with-rollers.png)
+![Modules without and with rollers](images/hepa-comparison.png)
```
Ah, so I was actually thinking of doing something like this:

```
<figure class="side-by-side" markdown>
![Module with door rollers](images/hepa-with-rollers.png)
![Module without door rollers](images/hepa-without-rollers.png)
<figcaption>Newer models have door rollers. Older models do not.</figcaption>
</figure>
```

jwwojak replied:
That's good too. I'm assuming this is a class rule that puts 2 separate images side by side. Will do that.

jwwojak committed:
Removing unused images and fix comparison images

Put the side by side comparisons in a CSS class rule.

ecormany commented on docs/hepa-uv/docs/product-specs.md:
```diff
@@ -83,3 +83,13 @@ The power consumption specifications are measured with the UV lights and fan on.
+
+## Door Protection Rollers
+
+With early model HEPA/UV Modules, fully opening the robot's front door could cause it to scrape against the top of the module. HEPA modules manufactured on or after July 2025 include three rollers mounted along its top-front edge. These rollers help prevent the HEPA module from damaging the front door when you open it.
```
Tiny changes. We have to remember the `/UV` because the HEPA module (no UV) is a different product entirely.

```suggestion
With early model HEPA/UV Modules, fully opening the robot's front door could cause it to scrape against the top of the module. HEPA/UV Modules manufactured in or after July 2025 include three rollers mounted along its top-front edge. These rollers help prevent the HEPA/UV Module from damaging the front door when you open it.
```

ecormany reviewed (approved):
Final proofreading change. GTG after that.

jwwojak committed:
Update docs/hepa-uv/docs/product-specs.md

Co-authored-by: Ed Cormany <edward.cormany@opentrons.com>
